src/main/java/net/neoforged/neoforge/registries/datamaps/builtin/NeoForgeDataMaps.java

```diff
@@ -106,5 +121,6 @@ private static void register(final RegisterDataMapTypesEvent event) {
         event.register(PARROT_IMITATIONS);
         event.register(RAID_HERO_GIFTS);
         event.register(VIBRATION_FREQUENCIES);
+        event.register(MONSTER_ROOM_MOBS);
```

Comment:
This was not moved as requested.

Reply:
Ignore this; I made the review at the same time as Matyrobbrt and they made the same comment before me.